cloud/linode/client/client.go

```diff
@@ -48,14 +57,21 @@ type Client interface {
 var _ Client = (*linodego.Client)(nil)
 
 // New creates a new linode client with a given token, userAgent, and API URL
-func New(token, userAgent, apiURL string) (*linodego.Client, error) {
-	linodeClient := linodego.NewClient(nil)
+func New(token, userAgent, apiURL string, timeout time.Duration) (*linodego.Client, error) {
```

Comment:
:question: Is it necessary to pass in `userAgent` and `apiURL`, or could those be safely defaulted?

If the use-case for specifying these is to override them in testing scenarios, that should be a separate concern that is handled in the testing code.

Reply:
No, they can be safely defaulted. Updated the PR to use default values.

Reply:
Sweet! Thank you.